sem_covid/entrypoints/etl_dags/crawl_pipeline.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, store_registry: StoreRegistryABC, file_name: str, bucket_name
         self.scrapy_crawler = scrapy_crawler
 
     def get_steps(self) -> list:
-        return [self.extract, self.transform_structure,
+        return [self.extract, self.transform_content,
```

Comment:
we shall call the transform structure, even if it is empty. 
All ETL steps shall be executed.

Reply:
better inherit this method from the Base.